app/code/core/Mage/Core/Model/Email/Template/Abstract.php

```diff
@@ -117,7 +117,7 @@ protected function _getLogoUrl($store)
                 return Mage::getBaseUrl('media') . $uploadDir . '/' . $fileName;
             }
         }
-        return Mage::getDesign()->getSkinUrl('images/logo_email.gif');
+        return Mage::getDesign()->getSkinUrl('images/logo.svg');
```

Comment:
Would this work?

```suggestion
        $fullFileName = Mage::getDesign()->getFilename('images/logo_email.gif', ['_type' => 'skin']);
        return file_exists($fullFileName)
             ? Mage::getDesign()->getSkinUrl('images/logo_email.gif');
             : Mage::getDesign()->getSkinUrl('images/logo.svg');
```

Reply:
I tested your code and it's great as usual (there's a `;` on the second line which shouldn't be there but it's no big deal).

The thing I'm wondering is... also using composer installation of a OM project, no file ever gets deleted from the file system also if it's deleted on our repo. which means that logo_email.gif will always be there...

This is a pickle.

Reply:
if we could remove the `base` folder from the fallback...